docs/admin/node.md

```diff
@@ -247,6 +248,12 @@ Set the `cpu` and `memory` values to the amount of resources you want to reserve
 Place the file in the manifest directory (`--config=DIR` flag of kubelet).  Do this
 on each kubelet where you want to reserve resources.
 
+## API Object
+
+Node is a top-level resource in the kubernetes REST API. More details about the
+API object can be found at: [Node API
+object](https://htmlpreview.github.io/?https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api-reference/definitions.html#_v1_node).
```

Comment:
Yes. It would have been great if https://htmlpreview.github.io/?http://releases.k8s.io/HEAD/docs/api-reference/definitions.html#_v1_node worked, but it does not.